src/Kamino.ts

```diff
@@ -1413,31 +1413,59 @@ export class Kamino {
       {}
     );
 
-    const raydiumStrategies = strategiesWithAddresses.filter(
-      (x) => x.strategy.strategyDex.toNumber() === dexToNumber('RAYDIUM') && x.strategy.position !== DEFAULT_PUBLIC_KEY
+    const activeStrategies = strategiesWithAddresses.filter(
+      (x) => x.strategy.position !== DEFAULT_PUBLIC_KEY
     );
-    const raydiumPoolsPromise = this.getRaydiumPools(raydiumStrategies.map((x) => x.strategy.pool));
-    const raydiumPositionsPromise = this.getRaydiumPositions(raydiumStrategies.map((x) => x.strategy.position));
-    const orcaStrategies = strategiesWithAddresses.filter(
-      (x) => x.strategy.strategyDex.toNumber() === dexToNumber('ORCA') && x.strategy.position !== DEFAULT_PUBLIC_KEY
+
+    const all = await batchFetch([...activeStrategies.map((x) => x.strategy.pool), ...activeStrategies.map((x) => x.strategy.position)], chunk => fetchEncodedAccounts(this._rpc, chunk));
```

Comment:
how does `batchFetch` work if we have duplicate keys, will it return duplicate data? Can you please test this behavior?